### PR TITLE
Template filter

### PR DESCRIPTION
**Pull request by Oddant1**

### CODE REVIEW

File: q2galaxy/core/templaters/helpers.py [hunk 1]
```diff
@@ -194,12 +197,16 @@ def inputs_xml(self):
 
         return param
 
+    def strip_pred(self, expr):
+        return expr.duplicate(fields=tuple(self.strip_pred(c) for c in expr.fields),
+                              predicate=IntersectionExp())
+
     def _make_validator(self):
         _validator_set = repr(set(map(str, self.qiime_type)))
```
Comment (ebolyen) [inline q2galaxy/core/templaters/helpers.py:205]:
```suggestion
        _validator_set = repr(set(map(str, map(strip_pred, self.qiime_type))))
```

**Comment by ebolyen** [comment]
@Oddant1, looks like there's a union sneaking in and breaking duplicate. I suspect it's something like Foo[Bar | Baz], but I'm not exactly sure why this wasn't unpacked by the outer loops where we use duplicate...

**Comment by lizgehret** [comment]
Hey @Oddant1 @ebolyen is this still current?

**Comment by Oddant1** [comment]
@lizgehret I'm pretty sure we don't need this anymore, but I'll defer to @ebolyen

**Comment by lizgehret** [comment]
@ebolyen to create issue to track this, will close out this PR and work on development elsewhere.

**Pull request closed without merging**